content/reference/sql/version-control/dolt-sql-functions.md

```diff
@@ -98,41 +101,43 @@ mysql> select dolt_version();
 
 Table functions operate like regular SQL functions, but instead of returning a single, scalar value, a
 table function returns rows of data, just like a table. Dolt's table function support is currently limited
-to only the `DOLT_DIFF()` function and table functions have several restrictions in how they can be used in 
+to only the `DOLT_DIFF()` function and table functions have several restrictions in how they can be used in
```

Comment:
"Dolt's table function support is currently limited to only the `DOLT_DIFF()` function" is out of date – we've got three whole table functions now! 😁 

Could you fix that line while you're in here?